### ISSUE TITLE
bug: misc accessibility issues

### ISSUE THREAD

**Issue by jlp-craigmorten** [guidepup/guidepup.dev#40]
From https://assistivlabs.com/articles/automating-screen-readers-for-accessibility-testing:

> The test detects multiple accessibility issues that previously slipped through:
>
> - Links to external sites are indicated by an icon which is available visually, but not programmatically/accessibly (1.3.1 failure)
> - The navigation region has an accessible name of “Main” and is followed by the main region, which could be confusing (best practice)
> - Invoking the “Get Started” link doesn’t cause NVDA to announce the new page (4.1.3 failure)

**Comment by WestonThayer** [comment]
Thanks for filing, I could've done you that courtesy 🙇🏻. I was assuming these are all in Docusaurus:

- https://github.com/facebook/docusaurus/issues/10654 was closest, I proposed a fix over in https://github.com/facebook/docusaurus/pull/11331
- Looks like `theme.NavBar.navAriaLabel` controls this. When you're on https://docusaurus.io/blog, there are several `<nav>`s, so seems reasonable they give each a label. Is `"Main"` a confusing label for the top one? I dunno, need user research. Seems super minor
- Opened https://github.com/facebook/docusaurus/issues/11332

**Comment by jlp-craigmorten** [comment]
Thanks for all the triage @WestonThayer! And dw, you don't owe anything, great enough that got some free QA out of you 😄
